README.md

```diff
@@ -1,8 +1,35 @@
-![Build Status](https://forcedotcom.github.io/SalesforceMobileSDK-TestResults/CordovaPlugin-results/dev/latest/buildstatus.svg)
+# Cordova Plugin for Salesforce Mobile SDK
```

Comment:
@rwhitley, if you have time, could you review the README.
It will appear on npmjs once we start publishing the plugin there.